raiden/transfer/channel.py

```diff
@@ -1705,6 +1713,10 @@ def handle_channel_closed(
         )
         if call_update:
             expiration = state_change.block_number + channel_state.settle_timeout
+            # silence mypy: checked above already
+            assert balance_proof is not None
+            # silence mypy: partner's balance proof is always signed
+            assert not isinstance(balance_proof, BalanceProofUnsignedState)
```

Comment:
As per my original comment please have a positive assertion (That this is a `BalaneProofSignedState`).